response_operations_ui/forms.py

```diff
@@ -249,3 +249,43 @@ def validate_password(form, field):
         if password.isalnum() or not any(char.isupper() for char in password) or not any(char.isdigit() for char in
                                                                                          password):
             raise ValidationError('Your password doesn\'t meet the requirements')
+
+
+class RequestAccountForm(FlaskForm):
+    '''The form used by request-new-account.html'''
+    email_address = StringField('Enter the ONS email address to create an account for',
+                                validators=[InputRequired('Enter an email address'),
+                                            Email(message='Invalid email address'),
+                                            Length(max=254,
+                                                   message='Your email must be less than 254 characters')])
+    password = PasswordField('Enter the admin password',
+                             validators=[InputRequired('Enter the admin password')])
+
+    @staticmethod
+    def validate_email_address(_, field):
+        email = field.data
+        _validate_email_address(email)
+        local_part, domain_part = email.rsplit('@', 1)
+        if domain_part not in ['ons.gov.uk', 'ext.ons.gov.uk', 'ons.fake']:
```

Comment:
Cant decide if this should be a config item or not . Would a non ons email be useful during testing ? If so then may be better as a config item ?

Reply:
I don't think this needs to be config for this we only want certain domains so this is fine.